### ISSUE TITLE
When downloading python, the speed is too slow

### ISSUE THREAD

**Issue by zackzhy** [replicate/cog#1401]
Cog downloads python from https://www.python.org/ by default. I know it is best for most users, but for me, the speed is too slow because of the area I'm in. Is there any way to change the download address?
![image](https://github.com/replicate/cog/assets/58976459/73206d89-6e5c-4760-8a28-b6ec3e5e236a)

**Comment by technillogue** [comment]
No, sorry, but you can try --use-cuda-base-image=false to use a base image that already has python installed. I also recommend using a remote server in GCP, AWS, or similar that has decent bandwidth

**Comment by markphelps** [comment]
Closing this as stale since there hasn't been any activity for over 2 years. As mentioned previously, there currently isn't a way to change the default Python download mirror, but you can avoid the download entirely by using a base image that already includes Python (e.g., with --use-cuda-base-image=false). If you're still running into issues, feel free to open a new issue.
